src/services/__mocks__/augurjs.js

```diff
@@ -0,0 +1,80 @@
+const mockAugur = jest.genMockFromModule("augur.js");
+
+const BigNumber = require("bignumber.js");
+
+const ten = new BigNumber(10, 10);
+const decimals = new BigNumber(18, 10);
+const multiple = ten.exponentiatedBy(18);
+const universesData = [
+  {
+    universe: "0xGENESIS",
+    payout: [],
+    isInvalid: false,
+    numMarkets: 15,
+    supply: "1100000000000000000000000",
+    parentUniverse: "0x0000000000000000000000000000000000000000"
+  },
+  {
+    universe: "0xGENESIS_2",
+    payout: [],
+    isInvalid: false,
+    numMarkets: 0,
+    supply: "50000000000000000000000",
+    parentUniverse: "0x0000000000000000000000000000000000000000"
+  },
+  {
+    universe: "0xCHILD_1",
+    payout: [10000, 0],
+    isInvalid: false,
+    numMarkets: 400,
+    parentUniverse: "0xGENESIS"
+  }
+];
+
+mockAugur.constants = {
+  PRECISION: {
+    decimals: decimals.toNumber(),
+    limit: ten.dividedBy(multiple),
+    zero: new BigNumber(1, 10).dividedBy(multiple),
+    multiple
+  }
+};
+
+mockAugur.rpc = {
```

Comment:
This might not be applicable here but these return values being here will couple the expectation to this mock.

If this mock is used elsewhere then we need to figure out a way to set these value in the test.

Reply:
Yeah, I was going to ask you about that... I will look into this